### ISSUE TITLE
Spotify or YouTube Music Support

### ISSUE THREAD

**Issue by swarupbhanjachowdhury** [TeamNewPipe/NewPipeExtractor#614]
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

As the web version of Spotify and actual Youtube Music is available on the web, can we get support of any of them?

(Separate YouTube music service)

open.spotify.com
music.youtube.com

**Comment by kowith337** [comment]
I think YouTube Music are already supported, but not fully, e.g. view specific artists as channel-like view

Now you can search from Music by tap three dots menu next to search, then you can see `YouTube Music` search filter group.

**Comment by swarupbhanjachowdhury** [comment]
Yes i was wanted a homepage like YouTube music, with country based recommendations.

**Comment by jefferson018200306** [comment]
How about the Spotify support, has it been already in the update?

**Comment by yoshimo** [comment]
There are no code fragments for Spotify, but you might as well use https://github.com/timdorr/spotifeed/
Both for reference and for proper rss feeds to your favourite podcast player.

**Comment by TiA4f8R** [comment]
Note that full Spotify support will be not possible beacause they are using DRMs on tracks.

**Comment by FireMasterK** [comment]
For Spotify, it would be worth using something like https://github.com/librespot-org/librespot-java or https://github.com/devgianlu/librespot-android

but careful consideration is required: https://github.com/librespot-org/librespot-java#disclaimer

**Comment by ikanakova** [comment]
Or something like this https://github.com/KRTirtho/spotube

**Comment by opusforlife2** [comment]
YT Music is already supported. Requests for a separate front-end have to go in the main repo, not here.
